mender-convert-modify

```diff
@@ -109,20 +109,24 @@ if [ -z "$deb_distro" -o -z "$deb_codename" ]; then
     deb_codename="buster"
 fi
 
-if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
-
-    log_info "Installing Mender client version ${MENDER_CLIENT_VERSION}"
-
-    if [ "${MENDER_CLIENT_VERSION}" = "latest" ]; then
-        deb_name=$(deb_from_repo_dist_get "work/deb-packages" ${MENDER_APT_REPO_URL} ${deb_arch} "${deb_distro}/${deb_codename}/stable" "mender-client")
-    elif [ "${MENDER_CLIENT_VERSION}" = "master" ]; then
-        deb_name=$(deb_from_repo_dist_get "work/deb-packages" ${MENDER_APT_REPO_URL} ${deb_arch} "${deb_distro}/${deb_codename}/experimental" "mender-client")
+function install_deb_pacakge() {
```

Comment:
Nice!
How about moving it to a module also?

Reply:
Okey. Good.